frontend/tsconfig.json

```diff
@@ -2,7 +2,7 @@
   "compilerOptions": {
     "outDir": "./dist/",
     "module": "esnext",
-    "target": "es6",
+    "target": "ES2018",
```

Comment:
Why is the target change necessary?

Reply:
This was because named capturing groups are only available in ES2018 and later, the regex used in `detectLinks` uses one. I could rewrite it to not use one, but I think this is cleaner, and all of the features newly available in ES2018 are baseline/widely available https://caniuse.com/sr_es9